### ISSUE TITLE
Fee as a %?

### ISSUE THREAD

**Issue by zhmrab** [cryptonotefoundation/cryptonote#157]
Is there any way for the fee to be a % instead of a flat value?

**Comment by jared201** [comment]
yes, but you have to do external computation from your web service and use the result to enter the actual fee to the transfer() or sendTransfer() method

**Comment by SoraKohaku** [comment]
percentage used decimal or so calculate by math. look some peer code. they
used %
